metals/src/main/scala/scala/meta/internal/bsp/BspConfigGenerationStatus.scala

```diff
@@ -7,15 +7,16 @@ object BspConfigGenerationStatus {
   type Message = String
   sealed trait BspConfigGenerationStatus
   case object Generated extends BspConfigGenerationStatus
-  case object Cancelled extends BspConfigGenerationStatus
+  case object Cancelled
```

Comment:
This is a bug that is already fixed, so we might need to update after the final release.